### PR TITLE
Adjust to rc1

### PR DESCRIPTION
**Pull request by moonglum**
Adjust the example to the current version of faucet-pipeline.

* [x] Adjust to the new, shorter version of the manifest configuration
* [x] Put in the rc candidates of all dependencies

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ gem 'puma', '~> 3.7'
 gem 'jbuilder', '~> 2.5'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-gem 'faucet_pipeline_rails'
+gem 'faucet_pipeline_rails', path: "/home/vagrant/Code/faucet_pipeline_rails"
```
Comment (FND) [inline Gemfile]: wat?

File: faucet.config.js [hunk 2]
```diff
@@ -1,46 +1,28 @@
...
 module.exports = {
   js: jsConfig,
   sass: sassConfig,
   static: staticConfig,
+  manifest: {
+    file: "./public/assets/manifest.json",
+    key: (f, targetDir) => path.relative(targetDir, f),
+    value: f => `/${path.relative("public", f)}`
+  },
   watchDirs: ["app/assets"]
```
Comment (FND) [inline faucet.config.js]: FWIW, I like to keep `watchDirs` at the very top of the config object.

I'm also not sure whether it's still worth using separate variables here; I'd just inline `*Config`.

File: package.json [hunk 3]
```diff
@@ -7,16 +7,15 @@
...
   "author": "Lucas Dohmen <lucas@dohmen.io> (https://lucas.dohmen.io)",
   "license": "MIT",
   "dependencies": {
-    "faucet-pipeline": "^0.13.0",
-    "faucet-pipeline-js": "^0.15.4",
-    "faucet-pipeline-sass": "^0.7.1",
-    "faucet-pipeline-static": "^0.4.0",
+    "faucet-pipeline-js": "^1.0.0-beta.1",
+    "faucet-pipeline-sass": "^1.0.0-beta.0",
+    "faucet-pipeline-static": "^1.0.0-beta.0",
```
Comment (FND) [inline package.json]: `"~1.0.0-rc.1"`

File: Gemfile (hunk 1 above)
Comment (moonglum) [reply Gemfile]: This was for local testing while it was unreleased. It is gone now.

File: faucet.config.js (hunk 2 above)
Comment (moonglum) [reply faucet.config.js]: Done. Except the watchDirs. Because I don't know why 🤔

File: package.json (hunk 3 above)
Comment (moonglum) [reply package.json]: Done

**Pull request merged**